inflammation/models.py

```diff
@@ -31,3 +31,14 @@ def daily_max(data):
 def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
+
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    number = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - number) * (entry - number))
+
+    s_dev2 = sum(devs) / len(data)
+    return {'standard deviation': s_dev2}
```

Comment:
Returning a dict doesn't match what is returned for all of the other functions in this module. We should probably switch to just returning the calculated value.

Reply:
And it mixes in responsibilites that should be with the view